### PR TITLE
✨Source CoinMarketCap: Make Connector Compatible with Builder

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
* Make the connector compatible with the builder

## How
* Import schema to manifest
* Update docs, 
* Completely made compatible with the builder

## User Impact
* No impact, chore for maintainability.

## Can this PR be safely reverted and rolled back?
- [x] YES 💚

### CODE REVIEW

**Comment by btkcodedev** [comment]
GitHub CI Green: Ready to merge :white_check_mark: 

image

**Comment by btkcodedev** [comment]
Removed $parameter macro
Migrated Schema
Migrated Spec
Migrated to Poetry flow,
GitHub CI Green

File: docs/integrations/sources/coinmarketcap.md [hunk 1]
```diff
@@ -36,5 +36,6 @@ Coinmarketcap APIs are under rate limits for the number of API calls allowed per
 
 | Version | Date       | Pull Request                                              | Subject                                         |
 | :------ | :--------- | :-------------------------------------------------------- | :---------------------------------------------- |
-| 0.1.1   | 2022-11-01 | [#18790](https://github.com/airbytehq/airbyte/pull/18790) | Correct coinmarket spec                         |
-| 0.1.0   | 2022-10-29 | [#18565](https://github.com/airbytehq/airbyte/pull/18565) | 🎉 New Source: Coinmarketcap API [low-code CDK] |
+| 0.1.2   | 2024-05-30 | [38134](https://github.com/airbytehq/airbyte/pull/38134)  | Make connector compatabile with builder         |
```
Comment (girarda) [inline docs/integrations/sources/coinmarketcap.md:39]:
```suggestion
| 0.1.2   | 2024-05-13 | [38134](https://github.com/airbytehq/airbyte/pull/38134)  | Make connector compatabile with builder         |
```

**Review by girarda — APPROVED** [review]
approved pending fixing the date in the changelog ✅

**Comment by btkcodedev** [comment]
Date fixed

**Pull request merged**